app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -69,6 +71,13 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
     }
   };
 
+  var getRemoveResourcesTypes = function() {
+    if (angular.isUndefined(vm.catalogItemModel.retirement_playbook_id) || vm.catalogItemModel.retirement_playbook_id === '')
+      vm.catalogItemModel['remove_resources_types'] = {"No": "no", "Yes": "yes"};
+    else
+      vm.catalogItemModel['remove_resources_types'] = {"No": "no", "Pre": "pre", "Post": "post"};
```

Comment:
with this group the value will be "no_with_playbook", "pre_with_playbook" and "post_with_playbook"